dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/EuTaxonomyPanelNonFinancials.vue

```diff
@@ -22,8 +22,8 @@
                   None: low level of comfort that the subject matter is not materially misstated."
         />
       </div>
-      <div class="col-12 text-left pb-0">
-        <h3>Revenue</h3>
+      <div class="card-section-title">
```

Comment:
Now the captions somehow seem less bold than in the figma. Can you have another look at this?

Reply:
Ok